drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -129,6 +129,23 @@ class DragAndDropBlock(
         enforce_type=True,
     )
 
+    item_size = String(
+        display_name=_("Item size"),
+        help=_(
+            "Standard size: makes items square or rectangular based on "
+            "the number of characters in an item. "
+            "Fit to size: makes items fit to text content of an item, also "
+            "supports custom width option for item sizing."
```

Comment:
I think these two options are a bit confusing. Both of them seem to fit to the text content, so what is the difference?

While I understand the difference here, because I have been part of these discussions, I think for a course author these two options would be a bit confusing.